src/components/cesium_map/api/spatial.js

```diff
@@ -300,6 +300,15 @@ export class ISamplesSpatial {
       destination: place.destination,
       orientation: place.orientation
     });
+    // update the camera position
+    const updatedSpatialView = new SpatialView(
+      place.viewDict.longitude,
+      place.viewDict.latitude,
+      place.viewDict.height,
+      place.viewDict.heading,
+      place.viewDict.pitch
+    )
+    this.viewer.camera.setView(updatedSpatialView.getView)
   }
```

Comment:
Update the camera position as well as we need to use the camera position to calculate the bounding box.